gh2/csv.py

```diff
@@ -59,8 +59,11 @@ def make_parser():
         action='append', dest='filter_labels', default=[],
     )
     args.add_argument(
-        'repository',
-        help='Repository to retrieve issues from (e.g., rcbops/rpc-openstack)',
+        '--repo',
+        help='Repository to retrieve issues from (e.g., rcbops/rpc-openstack). '
```

Comment:
You might want to run pep8 on the file, there are a few things to clean up such as these lines which are too long.